### PR TITLE
New department fields

### PR DESCRIPTION
**Pull request by j-jasik**
## Description

Add new department fields including Home department and Other departments for staff and faculty.

## Size and Type of change

- Small Change - 1 person needs to review this
- New feature
- Cache clear needed

## How Has This Been Tested?

Please briefly describe the tests that you ran to verify your changes.

locally everything looks good, and after a cache clear it looked good on XP too.

## Checklist:

Only check what applies and what you have done.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have tested on multiple browsers (Chrome, Firefox, Safari, IE suite)

### CODE REVIEW

File: app/db/db_functions.py [hunk 1]
```diff
@@ -111,11 +112,21 @@ def get_directory_data():
...
             bu_role_data = sorted(bu_role_list, key=lambda x: bu_role_sort_key[x])
             bu_role = bu_role_data
-            department = get_splits(data[item]['dept'])
             major = get_splits(data[item]['stu_majr'])
             minor = get_splits(data[item]['stu_minr'])
             college = get_splits(data[item]['stu_coll'])
+
+            # TODO: these are going to be old
             title = get_splits(data[item]['title'])
+            department = get_splits(data[item]['dept'])
```
Comment (c-schwarze) [inline app/db/db_functions.py]: these 3 lines can be deleted, as they are replaced by the staff/faculty ones

File: app/db/db_functions.py [hunk 2]
```diff
@@ -131,17 +142,23 @@ def get_directory_data():
                             'addr_state': addr_state,
                             'addr_street1': addr_street1,
                             'addr_street2': addr_street2,
+                            'addr_street3': addr_street3,
                             'role': bu_role,
-                            'department': department,
+                            # 'department': department,
```
Comment (c-schwarze) [inline app/db/db_functions.py]: delete

File: app/db/db_functions.py [hunk 3]
```diff
@@ -131,17 +142,23 @@ def get_directory_data():
...
                             'major': major,
                             'minor': minor,
                             'college': college,
-                            'title': title,
+                            # 'title': title,
```
Comment (c-schwarze) [inline app/db/db_functions.py]: delete

File: app/db/db_functions.py (hunk 1 above)
Comment (j-jasik) [reply app/db/db_functions.py]: 👍

File: app/db/db_functions.py (hunk 2 above)
Comment (j-jasik) [reply app/db/db_functions.py]: 👍

File: app/db/db_functions.py (hunk 3 above)
Comment (j-jasik) [reply app/db/db_functions.py]: 👍

**Pull request merged**